_posts/2020-05-02-bash-on-Windows

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: Executer du code C depuis Windows sans passer par Virtualbox
+author: Andru Onciul
+---
+
+#Installer Ubuntu 
+
+1.	Installer Ubuntu depuis le Windows Store 
+![Publish]({{ site.baseurl }}/images/UbuntuWindowsStore.PNG)
+
+1.1. Si cette erreur apparait: 
+![Publish]({{ site.baseurl }}/images/ErreurSubSystemComponent.PNG)
```

Comment:
Idem que ci-dessus. Aussi je trouve qu'une image n'est pas nécessaire ici, un bout de code peut faire l'affaire genre
```
Installing, this may take a few time...
```